src/pe/utils.rs

```diff
@@ -74,3 +76,24 @@ pub fn try_name<'a>(bytes: &'a [u8], rva: usize, sections: &[section_table::Sect
         }
     }
 }
+
+pub fn get_data<'a, T>(bytes: &'a [u8], sections: &[section_table::SectionTable], directory: &DataDirectory, file_alignment: u32) -> Result<T, DataDirectoryConversionError>
```

Comment:
This seems like it belongs as a method in DataDirectories ?

Reply:
I'm not sure, because it belongs to SectionTable as well.. Utils is the good place for this imo